claim/claimstore.go

```diff
@@ -46,7 +47,7 @@ func (s Store) Save(claim Claim) error {
 func (s Store) Read(name string) (Claim, error) {
 	bytes, err := s.backingStore.Read(ItemType, name)
 	if err != nil {
-		if err == crud.ErrRecordDoesNotExist {
+		if strings.Contains(err.Error(), crud.ErrRecordDoesNotExist.Error()) {
```

Comment:
I am not using the errors.Cause function because not all errors are coming from that package. Unfortunately for example when errors are bubbled up from net/rpc (which plugins use in Porter), they get up being special rpc error struct and I need basic string matching instead of exact equality on the error struct to make this work.